docs/client/iOS/_manualExposures.mdx

```diff
@@ -0,0 +1,90 @@
+import CodeBlock from "@theme/CodeBlock";
+
+export const Snippets = {
+  // Gates
+  gateSnippet: (
+    <CodeBlock language="swift">
+      {`// Swift
+let result = Statsig.checkGateWithExposureLoggingDisabled("a_gate_name")
+\t
+// Objective C
+bool result = [Statsig checkGateWithExposureLoggingDisabled:@"a_gate_name"];
+`}
```

Comment:
huh, this breaks from the pattern of all the ones above, which actually have a tabbed view for swift and objective c.  You dont need to fix all of this in this PR, but it seems we could follow up with a `Snippets` that supports multiple languages